miner/worker.go

```diff
@@ -1091,9 +1091,6 @@ func (w *worker) commitNewWorkForORB(interrupt *int32, noempty bool, timestamp i
 		log.Error("Failed to prepare header for mining", "err", err)
 		return
 	}
-	if w.env.IsUserActivated && w.env.NumURBmined.Cmp(big.NewInt(0)) == 0 {
-		header.Difficulty.Add(header.Difficulty, big.NewInt(1))
-	}
```

Comment:
Where the first URB with difficulty + 1 is created?

Reply:
It caused build error since this version do not support UAF. So I deleted it for testing my code. It will added if you merge https://github.com/Onther-Tech/plasma-evm/pull/31. So don't worry about this. Additionally, there is no more `w.env.NumURBmined` field in new version.